### ISSUE TITLE
Auto complete is not working properly

### ISSUE THREAD

**Issue by viggo** [bring/hedwig#1546]
Tabbing does not work with auto complete and the "dropdown" moves all other content down.

This vs autocomplete on navbar search?

From a another issue regarding the same thing:
From Pia Wold Nilsen:
We have some accessibility issues with this component where the autocomplete suggestions are not accessible with keyboard. We had a problem like this earlier with the dropdown component, where we fixed the accessibility issue by using the native dropdown, but for the autocomplete component we don’t have an option to use native. Is this something that will be fixed for the autocomplete component?

**Comment by laugerud** [comment]
Made a copy of this issue to KP Trello
